### ISSUE TITLE
Fastlane changelogs too long

### ISSUE THREAD

**Issue by IzzySoft** [maxrave-dev/SimpMusic#175]
Fastlane changelogs are limited to 500 chars. Most of yours currently exceed that limit and hence are truncated:

```
! metadata/com.maxrave.simpmusic/en-US/changelogs/12.txt has 639 chars
! metadata/com.maxrave.simpmusic/vi-VN/changelogs/11.txt has 744 chars
! metadata/com.maxrave.simpmusic/vi-VN/changelogs/12.txt has 744 chars
```

Can you please take care to stay within the limits? Thanks in advance!

PS, for reference, here's my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688)

**Comment by maxrave-dev** [comment]
Thanks, because 11, 12 version code have too much new features, sorry for that

**Comment by IzzySoft** [comment]
I've understood that. It still looks weird, as it's cut off exactly at char 500, regardless whether that's even mid-word. Hint: in such cases, you could restrict it to the most important changes, and end the list with something like "full details can be found at {link}", linking to a full changelog (e.g. release notes). That way, most important changes are listed alongside the app description – and those interested can simply follow the link.

**Comment by maxrave-dev** [comment]
@IzzySoft SimpMusic which on IzzyOnDroid wasn't updated to v0.1.7. Still on v0.1.6. How to fix bro?

**Comment by IzzySoft** [comment]
There is no release for v0.1.7, so where should my updater pick the APK from? (having a vague idea) Oh well. Welcome to the Github-release-bug. There is a release, it's just not on the release page, and Github's API does not list it… I've picked the APK manually (having guessed the name by your comment I was able to find it), should show up with the next sync around 7 pm UTC then.

Just wondering: could it be you created that release, then removed it, and finally recreated it at the same commit? Only trying to figure if it matches the "known pattern" of this bug.

**Comment by maxrave-dev** [comment]
Yeah, i use github api but still get latest is v0.1.6. But already have v0.17 in release and tag. Fdroid is still updated to 0.1.7

**Comment by IzzySoft** [comment]
So had you created the v0.1.7 release before, and at some point dropped and recreated it at the very same commit? Because that is a known way to make this issue happen.

And yes, as F-Droid builds from source it checks out your repo and goes [by tags](https://github.com/maxrave-dev/SimpMusic/tags) – where you see 0.1.7 does exist. But there are no APK files at the tags, which is why my updater goes [by releases](https://github.com/maxrave-dev/SimpMusic/releases) – where you can clearly see 0.1.7 is missing. This is a bug with Github, and I've seen this with at least 5 projects until now. You could go ahead and file an issue with them, maybe if more people do that they'll finally check and fix it. Until now we've been ignored.

**Comment by maxrave-dev** [comment]
@IzzySoft Do u have any idea to make v0.1.7 appeared again in Github API, my users dont have update notification

**Comment by IzzySoft** [comment]
While that has resolved itself (sorry, the notification somehow did not reach me) – your changelog issue popped up again:

    ! metadata/com.maxrave.simpmusic/en-US/changelogs/17.txt has 547 chars
    ! metadata/com.maxrave.simpmusic/vi-VN/changelogs/17.txt has 642 chars

Would you mind addressing it – ideally in a way that it does not repeat? :wink:

**Comment by IzzySoft** [comment]
@maxrave-dev ^^ ?? I've now manually removed 2 lines from en-US and the entire vi-VN 17.txt to get rid of the monthly warnings from my quality checker. Would be nice if you could take care of this on your end as well – at least making sure it does not repeat over and again. Thanks in advance!

**Comment by maxrave-dev** [comment]
Oh, thanks bro, I will be more careful

**Comment by IzzySoft** [comment]
`<Terminator voice> "I'll be back!" </Terminator voice>`

* build/com.maxrave.simpmusic/metadata/en-US/changelogs/51.txt has 735 chars
* build/com.maxrave.simpmusic/metadata/en-US/changelogs/52.txt has 519 chars
* build/com.maxrave.simpmusic/metadata/vi-VN/changelogs/51.txt has 693 chars
* build/com.maxrave.simpmusic/metadata/vi-VN/changelogs/52.txt has 588 chars

Can you be even more careful, @maxrave-dev? 🙈 Thanks in advance!
